downstream/assemblies/platform/assembly-controller-api-filter.adoc

```diff
@@ -2,4 +2,8 @@
 
 = Filtering in the API
 
-Any collection is what the system calls a “queryset” and can be filtered using various operators.
+Any collection is what the system calls a "queryset" that you can filter by using various operators.
```

Comment:
This is a bit of a strange sentence.

Reply:
I agree, what about:

"A collection is known by the system as a "queryset". You can filter this by using various operators."

Reply:
That'll do!